ui/components/app/security-provider-banner-alert/blockaid-banner-alert/blockaid-banner-alert.test.js

```diff
@@ -7,6 +7,8 @@ import {
 } from '../../../../../shared/constants/security-provider';
 import BlockaidBannerAlert from '.';
 
+global.encodeURIComponent = jest.fn((str) => 'mockedEncodedString');
```

Comment:
This needs to be mocked because it was somehow returning different data on local and CI thereby causing snapshots to fail on CI but pass on local.